src/transformer/originalFormatTransformer.ts

```diff
@@ -183,7 +183,7 @@ const valueTransformer: {} | undefined = {
   radius: defaultValueTransformer,
   spacing: defaultValueTransformer,
   motion: motionValueTransformer,
-  opacity: opacityValuesTransformer // TOOD
+  opacity: opacityValuesTransformer
```

Comment:
Sorry, just noticed, this should be `opacityValueTransformer` (= opacityValue**~s~**Transformer, without the s)